test/e2e-rh-sdl/rapidast/far/03-assert.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: rapidast-job
+  namespace: rapidast-far
```

Comment:
If we don't do something specific to FAR in this PR and namespace, then I think it would be nicer to use `rapidast-medik8s namespace` as eventually, we plan to add rapidast scanning PR to all the Medik8s operators.